tests/models/test_pnfrag.py

```diff
@@ -8,6 +8,34 @@
 
 initialI = Place("I",1)
 
+expected_net_parsed_with_spaces = LabelledPetriNet(
```

Comment:
This variable is only used once so please inline with the relevant test to match usage in the rest of the module

Reply:
Sure.